debug_toolbar/static/debug_toolbar/css/toolbar.css

```diff
@@ -220,7 +220,7 @@
     background-color: #fff;
     border: 1px solid #111;
     border-bottom: 0;
-    top: 0;
+    top: 268px;
```

Comment:
What do you think about using a relative position rather than an fixed value?

Reply:
Yes, that would work too. Let me try it out and see the changes

Reply:
or what if instead of changing the position, I adjust the y translation?
What do you think?